src/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -16,11 +16,7 @@ namespace System.Net.WebSockets.Client.Tests
 {
     public class SendReceiveTest : ClientWebSocketTestBase
     {
-        // Windows 10 Insider Preview Build 16215 introduced the necessary APIs for the UAP version of
-        // ClientWebSocket.ReceiveAsync to consume partial message data as it arrives, without having to wait
-        // for "end of message" to be signaled.
-        public static bool PartialMessagesSupported =>
-            !PlatformDetection.IsUap || PlatformDetection.IsWindows10InsiderPreviewBuild16215OrGreater;
+        public static bool PartialMessagesSupported { get; } = PlatformDetection.ClientWebSocketPartialMessagesSupported;
```

Comment:
There's a significant difference between:
```C#
SomeType SomeProperty { get; } = Foo;
```
and:
```C#
SomeType SomeProperty => Foo;
```
The former is equivalent to:
```C#
private readonly SomeType _somePropertyBackingField = Foo;
public SomeType SomeProperty { get { return _somePropertyBackingField; } }
```
and the latter is equivalent to:
```C#
public SomeType SomeProperty { get { return Foo; } }
```
so they're not interchangeable.